README.md

```diff
@@ -63,13 +63,20 @@ default_cluster: "YOUR_CLUSTER"
 ```
 
 ## Scheduling a job
-You are now ready to schedule jobs!
+You are now ready to schedule jobs! To run the following examples, make sure to install all dependencies:
+
+```bash
+pip install -e ".[extra]"
+```
+
```

Comment:
Given that we are describing the installation steps at l23, could we avoid duplicating them here? If this is just about setting extra by default, perhaps we can have this suggestion in the install instruction at l23 instead?

Reply:
yes makes sense. The code snippet below didn't run on my side without installing these extra dependencies